### DISCUSSION TITLE
antd5+formily2.2.14报错

### DISCUSSION THREAD

**Discussion by ffhope** [alibaba/formily#3617] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/48660931/206651343-6ec94c66-5548-4c52-8138-b9863559f616.png)

**Comment by theashishmaurya** [comment]
I am having the exact same issue 👍. Please let me know if you find any solution, this issue is occurring while using @formily/antd  package.

**Reply by dayuy** [reply]
pnpm i @formily/antd-v5 -P

**Comment by yiyunwan** [comment]
使用alias把～去掉

**Comment by ZYQiang** [comment]
![image](https://github.com/alibaba/formily/assets/21325933/8d6e6a4c-e879-4466-abb3-b5d801ef3a3f)

我准备按照官网教程  好多报错，无法继续
